crates/amf0/src/de/mod.rs

```diff
@@ -42,12 +42,57 @@ where
     Ok(value)
 }
 
+#[cfg(feature = "serde")]
+#[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
```

Comment:
We dont need `doc(cfg)` because we have a crate feature `#![cfg_attr(docsrs, feature(doc_auto_cfg))]`